test/cppsuite/tests/schema_sweep.cpp

```diff
@@ -0,0 +1,95 @@
+/*-
+ * Public Domain 2014-present MongoDB, Inc.
+ * Public Domain 2008-2014 WiredTiger, Inc.
+ *
+ * This is free and unencumbered software released into the public domain.
+ *
+ * Anyone is free to copy, modify, publish, use, compile, sell, or
+ * distribute this software, either in source code form or as a compiled
+ * binary, for any purpose, commercial or non-commercial, and by any
+ * means.
+ *
+ * In jurisdictions that recognize copyright laws, the author or authors
+ * of this software dedicate any and all copyright interest in the
+ * software to the public domain. We make this dedication for the benefit
+ * of the public at large and to the detriment of our heirs and
+ * successors. We intend this dedication to be an overt act of
+ * relinquishment in perpetuity of all present and future rights to this
+ * software under copyright law.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+ * OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ * ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ * OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include "src/common/constants.h"
+#include "src/common/logger.h"
+#include "src/main/test.h"
+
+namespace test_harness {
+
+/*
+ * The intent of the test is to stress the code paths related to schema operations and dhandles
+ * management. This test keeps performing schema operations while reconfiguring the sweep server.
+ */
+class schema_sweep : public test {
+public:
+    schema_sweep(const test_args &args) : test(args)
+    {
+        init_operation_tracker();
+    }
+
+    /* Reconfigures the connection with different sweep server parameters 50% of the time. */
+    void
+    custom_operation(thread_worker *tw) override final
+    {
+        WT_CONNECTION *conn = connection_manager::instance().get_connection();
+        bool aggressive_sweep = false;
+        const std::string aggressive_sweep_cfg =
+          "file_manager=(close_handle_minimum=0,close_idle_time=1,close_scan_interval=1)";
+        const std::string default_sweep_cfg =
+          "file_manager=(close_handle_minimum=250,close_idle_time=30,close_scan_interval=10)";
+
+        while (tw->running()) {
+            tw->sleep();
```

Comment:
nit: Our `schema_sweep_default.txt` config runs for 5 seconds but sets `custom_config::op_rate` to 30 seconds, so by the time this sleep ends we're already tearing down the test. 
Do we need this default config or would cppsuite let us only provide the `_stress.txt` config?

Reply:
I would prefer having only a `_stress.txt` config but there is a test that executes `./run` and this starts every test with their default configuration.